configure-ironic.sh

```diff
@@ -13,6 +13,20 @@ IRONIC_FAST_TRACK=${IRONIC_FAST_TRACK:-true}
 # Whether cleaning disks before and after deployment
 IRONIC_AUTOMATED_CLEAN=${IRONIC_AUTOMATED_CLEAN:-true}
 
+# Timeout (seconds) to wait for a callback from a deploy ramdisk
+IRONIC_DEPLOY_CALLBACK_TIMEOUT=${IRONIC_DEPLOY_CALLBACK_TIMEOUT:-"4800"}
```

Comment:
the options look good, just the values seem a bit higher than expected, how did you get them?

Reply:
@elfosardo I was checking [ironic.conf file](https://github.com/metal3-io/ironic-image/blob/e1d39417015f5a7799a42a91a36f7005d872b562/ironic.conf#L31) and the values were set in that file. I guess it sets this value in purpose to make sure [boot_retry_timeout](https://github.com/metal3-io/ironic-image/blob/e1d39417015f5a7799a42a91a36f7005d872b562/ironic.conf#L56) is 3 times less than that variable as described in the comments. Maybe, we can set it to **"3600"** if it is higher than expected?

Reply:
I see, now I remember why we set the value like that. That's fine then, thanks!

Reply:
Thank you for reviewing.